CMakeBasePresets.json

```diff
@@ -163,6 +163,17 @@
                 "OPENDAQ_SKIP_FLAKY_TESTS": true
             }
         },
+        {
+            "name": "flaky_tests_ci",
+            "hidden": false,
+            "inherits": [
+                "ci"
+            ],
+            "cacheVariables": {
+                "OPENDAQ_ENABLE_FLAKY_TEST_LABELS": true,
+                "OPENDAQ_SKIP_FLAKY_TESTS": false
```

Comment:
Also... while we're at it, could we rename this to "OPENDAQ_SKIP_UNSTABLE_TESTS".